pkg/client/client.go

```diff
@@ -148,32 +149,33 @@ func (c *Client) SecretListWatchForNamespace(ns string) *cache.ListWatch {
 
 func (c *Client) WaitForPrometheusOperatorCRDsReady() error {
 	return wait.Poll(time.Second, time.Minute*5, func() (bool, error) {
-		err := c.WaitForCRDReady(k8sutil.NewCustomResourceDefinition(monv1.DefaultCrdKinds.Prometheus, mon.GroupName, map[string]string{}, false))
+		// TODO(paulfantom): re-add checking for CRDs
```

Comment:
to be removed?

Reply:
I removed waiting for CRDs as those are managed externally.